src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -261,8 +236,29 @@ static int32_t load_icu_data ()
     const char* bundle = get_bundle_path ();
     chdir (bundle);
 
+    char icu_dat_path [1024];
+    int res;
+
+    res = snprintf (icu_dat_path, sizeof (icu_dat_path) - 1, "%s/%s", bundle, "icudt.dat");
+    assert (res > 0);
+
     // TODO: set TRUSTED_PLATFORM_ASSEMBLIES, APP_PATHS and NATIVE_DLL_SEARCH_DIRECTORIES
-    monovm_initialize(0, NULL, NULL);
+    const char *appctx_keys [] = {
+        "RUNTIME_IDENTIFIER", 
+#ifndef INVARIANT_GLOBALIZATION
```

Comment:
nit: does the order matter? It might be a little more readable if the ifndef was at the end of the array declaration?

Reply:
The order does not matter from what I understand.